docs/biowulf.md

```diff
@@ -0,0 +1,28 @@
+# nf-core/configs: biowulf Configuration
+
+nf-core pipelines have been configured for use on the Biowulf cluster at the NIH.
+
+To use, run the pipeline with `-profile biowulf`, this will pull the [`biowulf.config`] (../conf/biowulf.config) from github which has been pre-configured with a setup suitable for the Biowulf cluster. By default, Nextflow will download Singularity containers to a cache directory setted by the config file. 
+
+```bash
+nextflow run <pipeline> -profile biowulf <Add your other parameters>
+
+```
+
+Before running the pipeline you will need to load Nextflow using the environment module system on Biowulf. You can do this by issuing the commands below:
+
+```bash
+## Load Nextflow which will also load Singularity environment modules
+module purge
+module load nextflow
+
+```
+
+The Illumina iGenomes resource is available on Biowulf. However it is different from the s3-hosted nf-core iGenomes. There are some structural differences of note. A local copy of the AWS-iGenomes resource is under consideration. In particular, if using BWA, the igenomes.conf should be modified to specify the BWA version folder, otherwise the BWA module will fail to find an appropriate index. To date, this is the only issue, however functionality has not been extensively tested with iGenomes on Biowulf. Nonetheless, you should, in theory, be able to run the pipeline against any reference available in the `igenomes.config` specific to the nf-core pipeline.
+
+You can do this by simply using the `--genome <GENOME_ID>` parameter.
+
+All of the intermediate files required to run the pipeline will be stored in the `work/` directory. It is recommended to delete this directory after the pipeline has finished successfully because it can get quite large, and all of the main output files will be saved in the `results/` directory anyway.
```

Comment:
You could consider adding `cleanup = true` within the config to auto remove the contents of `work/` once the pipeline completes, but depends how strict you want to be on this.

Reply:
I removed that part too due to that will break `-resume` function, which is pretty neat to me. But good to know there is option.